### DISCUSSION TITLE
RP2040 bitmaps

### DISCUSSION THREAD

**Discussion by emueyes** [Bodmer/TFT_eSPI#2518] [Raspberry Pi Pico]
How do I go about storing and reading a bitmap on the RP2040? Just copying the file to the root directory of the 2040 doesn't seem to work.

Edit: I should add, I'm using a display (GC9A01) that doesn't have an SD card reader.

Edit: I've now figured out that LittleFS is supported on the RP2040, and can list files copied to a LittleFS store on the RP2040. Actually displaying them on the TFT is still a problem for me..

**Comment by Bodmer** [comment]
You need to convert the file to an array or load it into LittleFS.

There are online tools to convert files to c arrays. These are mentioned with links in some of the examples.
